File: src/main/java/ru/nucodelabs/gem/view/control/Placeholder.java
package ru.nucodelabs.gem.view.control;

import javafx.beans.property.StringProperty;
import javafx.fxml.FXML;
import javafx.scene.control.Label;

public class Placeholder extends VBUserControl {
    @FXML
    private Label text;

    public String getText() {
        return text.textProperty().get();
    }

    public void setText(String text) {
        this.text.textProperty().set(text);
    }

    public StringProperty textProperty() {
        return text.textProperty();
    }
}

File: src/main/java/ru/nucodelabs/gem/view/control/Placeholder.kt
package ru.nucodelabs.gem.view.control

import javafx.beans.property.StringProperty
import javafx.fxml.FXML
import javafx.scene.control.Label

class Placeholder : VBUserControl() {

    @FXML
    private lateinit var text: Label

    var textValue: String
        get() = text.text
        set(value) {
            text.text = value
        }

    fun textProperty(): StringProperty = text.textProperty()
}
